bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml

```diff
@@ -80,7 +80,9 @@ spec:
         port: 6443
         path: readyz
       initialDelaySeconds: 10
```

Comment:
should we also exclude the etcd check from the livez path ?

Reply:
actually why not to make the probes to look exactly the same for the bootstrap server and a standard server ?